<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>发现音乐</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <property name="horizontalSpacing">
        <number>20</number>
       </property>
       <property name="verticalSpacing">
        <number>10</number>
       </property>
       <item row="2" column="2">
        <widget class="QGraphicsView" name="FindMusicgraphicsView6"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="FindMusicBtn3">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGraphicsView" name="FindMusicgraphicsView4"/>
       </item>
       <item row="0" column="2">
        <widget class="QGraphicsView" name="FindMusicgraphicsView3"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="FindMusicBtn1">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGraphicsView" name="FindMusicgraphicsView1"/>
       </item>
       <item row="2" column="1">
        <widget class="QGraphicsView" name="FindMusicgraphicsView5"/>
       </item>
       <item row="0" column="1">
        <widget class="QGraphicsView" name="FindMusicgraphicsView2"/>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="FindMusicBtn2">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="FindMusicBtn4">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="FindMusicBtn5">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="FindMusicBtn6">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
